src/qml/ListItem.qml

```diff
@@ -16,45 +17,72 @@
  */
 
 import QtQuick 2.9
+import QtGraphicalEffects 1.15
 import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
 
 Item {
     id: gridItem
+
     property alias title: label.text
     property alias iconName: icon.name
     signal clicked()
 
     width: parent.width
-    height: Dims.h(16)
-
-    Icon {
-        id: icon
-        width: parent.height
-        height: width
-        anchors.top: parent.top
-        anchors.left: parent.left
-        anchors.leftMargin: Dims.w(15)
-    }
-    Label {
-        id: label
-        font.pixelSize: Dims.l(6)
-        horizontalAlignment: Text.AlignHCenter
-        anchors.leftMargin: Dims.w(12)
-        anchors.left: icon.right
-        anchors.verticalCenter: parent.verticalCenter
-    }
+    height: Dims.h(21)
 
     MouseArea {
         id: mouseArea
+
         anchors.fill: parent
-        hoverEnabled: true
         onClicked: gridItem.clicked()
     }
 
     Rectangle {
+        id: highlightPress
+
         anchors.fill: parent
-        color: "white"
+        color: "#ffffff"
         opacity: mouseArea.containsPress ? 0.2 : 0
+        Behavior on opacity {
+            NumberAnimation {
+                duration: 150;
+                easing.type: Easing.Linear
+            }
+        }
+    }
+
+    Item {
+        id: shadowArea
```

Comment:
Mhh it's now a bit odd to name this shadowArea when there's no shadow :p but let's act cool and pretend this is about the highlight.